app/services/account_creation/device_profiling.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module AccountCreation
+  class DeviceProfiling
```

Comment:
The equivalent `Proofing::Resolution::ProgressiveProofer` is called within an asynchronous job so that it doesn't block the main application thread. Would we want to do something similar, especially since we're not concerned with awaiting the result?

Reply:
moved over to async job.